datahub-web-react/src/components/entity/dataset/profile/Ownership.tsx

```diff
@@ -1,12 +1,12 @@
-import { AutoComplete, Avatar, Button, Col, Row, Select, Table, Typography } from 'antd';
-import React, { useMemo, useState } from 'react';
+import { AutoComplete, Avatar, Button, Form, Select, Space, Table, Tag, Typography } from 'antd';
+import React, { useEffect, useMemo, useState } from 'react';
 import { Link } from 'react-router-dom';
-import { EntityType, Owner, OwnershipType, OwnershipUpdate } from '../../../../types.generated';
+import { EntityType, Owner, OwnershipSourceType, OwnershipType, OwnershipUpdate } from '../../../../types.generated';
 import defaultAvatar from '../../../../images/default_avatar.png';
 import { useGetAutoCompleteResultsLazyQuery } from '../../../../graphql/search.generated';
 import { useEntityRegistry } from '../../../useEntityRegistry';
 
-const OWNER_SEARCH_PLACEHOLDER = 'Enter an LDAP...';
+const OWNER_SEARCH_PLACEHOLDER = 'Search an LDAP';
```

Comment:
Content suggestion: Search for Owners  -> instead of LDAP

Reply:
_Techically_ we are search by LDAP ids. Is it clearer to write "owner" or "ldap" so they know specifically how to search?